scripts/README.md

```diff
@@ -15,7 +15,7 @@ Use once the final English version is already available in a JSON file in `/data
 Usage:
 
 ```
-node build/scripts/generate-spanish-descriptions.js <state_id>
+npx ts-node build/scripts/generate-spanish-descriptions.ts <state_id>
```

Comment:
I'm not sure if I'm missing something obvious here, but these are the scripts I had to run in order to run these TS files

Reply:
Hmm, I've never used `npx` to run scripts. The previous version is what I use. Can you make this additive (i.e. an alternative) rather than a replacement?

Reply:
if the previous command works for you, then I must be missing some compile step locally, because there is no `build` directory and no `.js` files in my local repo 🤔 I looked through the various scripts in `package.json` and none appear to create a `build` directory (though I'm now noticing that `yarn start` doesn't work for the same reason.

Reply:
Ah – the implicit assumption behind those commands is that you've compiled first. `yarn tsc` will create that `build/` directory.

I had assumed everyone was compiling one way or another (either with yarn tsc or having configured it to run automatically in one's code editor), and I guess the `yarn start` made the same assumption.

If you can add a note indicating that you should compile first for the `node` version, then I guess that would make the advice cover all approaches.

Reply:
ah, of course that's it! I searched through the code for `tsc` but didn't see it anywhere, so I had assumed that it wasn't expected. 🙃  thanks Dan